src/Symfony/Component/String/Inflector/InflectorInterface.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\String\Inflector;
+
+interface InflectorInterface
+{
+    /**
+     * Returns the singular forms of a string.
+     *
+     * If the method can't determine the form with certainty, several possible singulars are returned.
+     *
+     * @return string[] An array of possible singular forms
+     */
+    public function singularize(string $plural): array;
```

Comment:
I just renamed the args on the interface from $string to $plural and $singular, WDYT?